packages/data-grid/src/tableHeader/TableHeader.jsx

```diff
@@ -71,25 +87,44 @@ function TableHeader({
 }) {
   return (
     <div className="dg-table-header-container">
+      <div className="dg-table-header-content-container">
+        {search && (
+          <Input
```

Comment:
Can an `aria-label` attribute be added? `placeholder` is not reliably announced by assistive technology.